CoreQualityControlView/test/unit/src/au/gov/asd/tac/constellation/views/qualitycontrol/daemon/QualityControlAutoVetterNGTest.java

```diff
@@ -109,18 +108,22 @@ public void testInitWithRefresh() throws InterruptedException {
 
     // Test the adding and removing of observers and their behaviour to trigger the methods of the interface.
     @Test
-    public void testAddRemoveObserver() throws InterruptedException {
+    public void testAddRemoveObserver() {
         // add observer of the button state
-        final testObserver observer = new testObserver();
+        final TestObserver observer = new TestObserver();
         QualityControlAutoVetter.getInstance().addObserver(observer);
 
         // Check initial status
         assertFalse(observer.getCanRunStatus());
 
         QualityControlAutoVetter.updateQualityControlState(null);
 
-        // Sleep until after pluginExecution thread has returned
-        Thread.sleep(1000);
+        try {
+            // Sleep until after pluginExecution thread has returned
+            Thread.sleep(1000);
+        } catch (InterruptedException ex) {
```

Comment:
I reckon you should still be alright to throw the exception as you had before rather than catching here (similarly for other tests)